privacyidea/models.py

```diff
@@ -2174,7 +2174,7 @@ class SMSGatewayOption(MethodsMixin, db.Model):
     id = db.Column(db.Integer, Sequence("smsgwoption_seq"), primary_key=True)
     Key = db.Column(db.Unicode(255), nullable=False)
     Value = db.Column(db.UnicodeText(), default=u'')
-    Type = db.Column(db.Unicode(100), default=u'')
+    Type = db.Column(db.Unicode(100), default=u'option')
```

Comment:
Wouldn't adding a "default" in the models.py create a need for a new migrations script? Or is the default not reflected in the schema?